examples/hosts/app-integration/external-controller/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const path = require("path");
+const merge = require("webpack-merge");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+// const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+
+module.exports = env => {
+    const isProduction = env && env.production;
+
+    return merge({
+        entry: {
+            app: "./src/app.ts"
```

Comment:
Future question, but what is the container export now? Perhaps the registry? Unclear.

Reply:
E.g. historically we would have needed to export the container (fluidExport) & the app.

Reply:
Currently no export since we don't do any true code load with it, we could add one later if we want.